src/lib/config/arbitrum/pools.ts

```diff
@@ -142,6 +142,15 @@ const pools: Pools = {
     '0xd3d5d45f4edf82ba0dfaf061d230766032a10e07000200000000000000000413', // stg/ bb-a-usd
     '0x9fb7d6dcac7b6aa20108bad226c35b85a9e31b63000200000000000000000412', // wsteth-bb-a-weth / bb-a-usd
   ],
+  Deprecated: {
+    '0x178e029173417b1f9c8bc16dcec6f697bc323746000200000000000000000158': {
+      newPool:
+        '0x9fb7d6dcac7b6aa20108bad226c35b85a9e31b63000200000000000000000412 ',
+      description: 'deprecatedPool.migrateCardDescription',
+      joinsDisabled: true,
```

Comment:
Will this ever be false for deprecated pools? Same with stakingDisabled. If they'll always be true they don't need to go here.

Reply:
true, will remove this